src/Traits/ModifierTrait.php

```diff
@@ -243,12 +243,31 @@ public function second($value)
      * Add years to the instance. Positive $value travel forward while
      * negative $value travel into the past.
      *
+     * When adding or subtracting years, if the resulting time is a date
+     * that does not exist, the result of this operation will always be the
+     * last day of the intended month.
+     *
+     * ### Example:
+     *
+     * ```
+     *  (new Chronos('2015-01-03'))->addYears(1); // Results in 2016-01-03
+     *
+     *  (new Chronos('2012-02-29'))->addYears(1); // Results in 2013-02-28
+     * ```
+     *
      * @param int $value The number of years to add.
      * @return static
      */
     public function addYears($value)
     {
-        return $this->modify((int)$value . ' year');
+        $month = $this->month;
+        $date = $this->modify((int)$value . ' year');
+
+        if ($date->month !== $month) {
+            return $date->modify('last day of previous month');
+        }
```

Comment:
This changes the semantics of `addYears()` in a good way, but this and the interface changes will require a minor release bump.